docs/aichat/ai-service-integration.md

```
@@ -0,0 +1,521 @@
+    generate-response: "/ai-response"
+  timeout: 30000  # 30초
+  retry:
+    max-attempts: 3
+    delay: 1000   # 1초
+```
+
+### HTTP 클라이언트 설정
+```java
+@Configuration
+public class HttpClientConfig {
+    
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)